### PR TITLE
[zh]translation optimization

### PR DESCRIPTION
**Pull request by wangyysde**
Page： https://kubernetes.io/zh/docs/concepts/cluster-administration/system-metrics/
Some sentences on the above page was confused,i try to optimizate itt.

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/system-metrics.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ In most cases metrics are available on `/metrics` endpoint of the HTTP server. F
 
 Examples of those components:
 -->
-## Kubernetes 中的指标
+## Kubernetes 中组件的指标
 
-在大多数情况下，可以在 HTTP 服务器的 `/metrics` 端点上访问度量值。
-对于默认情况下不公开端点的组件，可以使用 `--bind-address` 标志启用。
+在大多数情况下，可以通过HTTP访问组件endpoint的 `/metrics` 路径来获取组件的度量值。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/system-metrics.md:42]:
```suggestion
在大多数情况下，可以通过 HTTP 访问组件的 `/metrics` 端点来获取组件的度量值。
```

File: content/zh/docs/concepts/cluster-administration/system-metrics.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ In most cases metrics are available on `/metrics` endpoint of the HTTP server. F
...
+对于那些默认情况下不暴露的endpoint的组件，可以使用 `--bind-address` 标志启用。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/system-metrics.md:43]:
```suggestion
对于那些默认情况下不暴露端点的组件，可以使用 `--bind-address` 标志启用。
```

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Review by tanjunchen — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**